tests/integration/test_cloudwatch.py

```diff
@@ -63,14 +64,7 @@ def test_put_metric_data_gzip(self):
         url = config.get_edge_url()
         headers = aws_stack.mock_aws_request_headers("cloudwatch")
 
-        authorization = (
-            "AWS4-HMAC-SHA256 Credential=test/20201230/"
-            "us-east-1/monitoring/aws4_request, "
-            "SignedHeaders=content-encoding;host;"
-            "x-amz-content-sha256;x-amz-date, Signature="
-            "bb31fc5f4e58040ede9ed751133fe"
-            "839668b27290bc1406b6ffadc4945c705dc"
-        )
+        authorization = mock_aws_request_headers("monitoring")["Authorization"]
```

Comment:
```suggestion
        authorization = aws_stack.mock_aws_request_headers("monitoring")["Authorization"]
```

just for consistency with L65